src/main/java/FileUtils.java

```diff
@@ -0,0 +1,49 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class FileUtils {
```

Comment:
I think this class is missing header comments, as well as method documentations.

Reply:
Thanks! I agree that it should have a header comment since it's a public class so I've added it